DEVELOP.rst

```diff
@@ -37,8 +37,6 @@ For help, run:
 
       check   Build, test, and lint the documentation
 
-      delint  Remove any `*.lint` files
```

Comment:
Why are we removing this line?

Reply:
I thought I left a comment here before. weird. anyway, the answer is: the lint files are hidden from the user now, so there's no need for us to expose a cleanup command anymore